internal/output/markdowntable.go

```diff
@@ -0,0 +1,63 @@
+package output
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/osv-scanner/internal/osv"
+	"github.com/google/osv-scanner/pkg/models"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+)
+
+// PrintTableResults prints the osv scan results into a human friendly table.
+func PrintMarkdownTableResults(vulnResult *models.VulnerabilityResults, outputWriter io.Writer) {
```

Comment:
Let's avoid duplicating the actual table construction, can you pull the table building logic out of both MarkdownTableResults and PrintTableResults into a private function, and have that function be called to generate the outputTable variable. 

I believe the only difference is one has `outputTable.RenderMarkdown()` and the other `outputTable.Render()`.

Reply:
Heading is different, issues and links are represented as links only and new lines must be represented as HTML tags "<br>"...

They are otherwise pretty similar.

If you want to have a crack at making a generic table generator that can handle this and any other possible future output formats (HTML, CSV, TSV, etc), that'd be cool...  I doubt I would be able to make such a function generic enough with my beginner level knowledge of go. 😅

Reply:
I had a go at merging some of the logic together, can you give it a quick test and if it's fine we should be able to merge this in!

Reply:
LGTM